project-base/storefront/components/Blocks/Skeleton/SkeletonPageProductsList.tsx

```diff
@@ -1,57 +1,11 @@
 import { SkeletonModuleBreadcrumbs } from './SkeletonModuleBreadcrumbs';
-import { SkeletonModuleProductListItem } from './SkeletonModuleProductListItem';
+import { SkeletonModuleProductsList } from './SkeletonModuleProductsList';
 import { Webline } from 'components/Layout/Webline/Webline';
-import { createEmptyArray } from 'helpers/arrayUtils';
-import Skeleton from 'react-loading-skeleton';
 
 export const SkeletonPageProductsList: FC = () => (
```

Comment:
You don't use this file anywhere. I think what you wanted to do is to use this within SkeletonManager, right?

Reply:
Ah, yes, I messed this up. This should be used in SkeletonManager. Refactored.